homeassistant/components/google_assistant/http.py

```diff
@@ -98,6 +104,16 @@ def should_report_state(self):
         """Return if states should be proactively reported."""
         return self._config.get(CONF_REPORT_STATE)
 
+    @property
+    def local_sdk_webhook_id(self):
+        """Return the local SDK webhook ID."""
+        return self._store.webhook_id
+
+    @property
+    def local_sdk_user_id(self):
+        """Return the user ID to be used for actions received via the local SDK."""
+        return list(self._store.agent_user_ids.keys())[0]
```

Comment:
This function needs to be updated to receive the webhook ID that received the actions and map it to the correct user.

Reply:
I think that the best way to approach this is to make the used user id a configuration for the component. This would support the use case as describe by 1. above. As far as I can tell the local sdk does not send this information and has no way of knowing which user agent id should be used otherwise. 

Having it be webhook dependent does not really make sense as we only send this webhook information for local fulfillment and we send it in a field that will get overwritten by the next sync with different data, i.e. the way we sync all devices for all user agent ids, to me look like we will only have the last synced value stored by google (as the device id stays the same).